lib/tasks/reviews.rake

```diff
@@ -0,0 +1,80 @@
+require 'json'
+require 'net/http'
+require 'optparse'
+require 'uri'
+require 'reverse_markdown'
+namespace :reviews do
+  desc 'Imports review from NRDBc, currently storing usernames as strings instead of references'
+
+  def retrieve_reviews
+    url = URI('https://netrunnerdb.com/api/2.0/public/reviews')
+
+    http = Net::HTTP.new(url.host, url.port)
+    http.use_ssl = true
+    request = Net::HTTP::Get.new(url)
+
+    response = http.request(request)
+
+    return JSON.parse(response.body) if response.is_a?(Net::HTTPSuccess)
+
+    raise "Failed to retrieve decklists! Status code: #{response.code}"
+  end
+
+  def purge_tables
+    # Only do this in a transaction
```

Comment:
you could assert that `ActiveRecord::Base.connection.transaction_open?` is true here.